xml/cap_admin_credhub.xml

```diff
@@ -40,11 +40,43 @@ cloudfoundry-incubator/credhub-cli
 </screen>
 </sect1>
 
+<sect1 xml:id="sec.cap.credhub-sizing">
+    <title>Enable the Credhub User</title>
+    <para>
+        You must enable the <literal>credhub</literal> user for your deployment
```

Comment:
We should just say refer to it as enabling CredHub and not CredHub user. The user suffix is there to differentiate between the CredHub for internal use by components and the user facing CredHub (what's being documented here) used as a service by users for their apps.